### PR TITLE
Fixing texts in the "encrypt wallet" GUI process

### PR DESCRIPTION
**Pull request by glaksmono**
Issue Reference: https://github.com/bitcoin/bitcoin/issues/13245

Updating some of the texts per suggestion in the issue.

### CODE REVIEW

File: src/qt/bitcoingui.cpp [hunk 1]
```diff
@@ -516,13 +516,13 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
         }
 #endif // ENABLE_WALLET
         unitDisplayControl->setOptionsModel(_clientModel->getOptionsModel());
-        
+
```
Comment (promag) [inline src/qt/bitcoingui.cpp]: nit, don't add unrelated changes just because your editor changed it, same below.

Comment (glaksmono) [reply src/qt/bitcoingui.cpp]: I put back the spaces, and I'm getting `lint` error from Travis:

https://travis-ci.org/bitcoin/bitcoin/jobs/384047372#L756

```
@@ -525 +525 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
+
^---- failure generated from contrib/devtools/lint-whitespace.sh
```
